charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.2.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.1.0
+version: 2.1.1
```

Comment:
I don't know that this is a bug fix, feels more like new functionality.  Seems like the chart version should go up a minor.

Reply:
Bumped minor instead, especially after adding the other sidecars.